src/views/NoteListView.vue

```diff
@@ -1,7 +1,104 @@
 <template>
-  <h1>Note List</h1>
+
+  <div class="search">
+  <div class="box">
+  <p>チャンネル:</p>
+  <input type="search" id="channel" name="search">
+  <button onclick="channel()" ><img src="../assets/検索_mock.png" width="25px" height="25px"/></button>
+  <input type="checkbox" checked>
+  <p>子チャンネルを含む</p>
+  </div>
+  <div class="box">
+  <p>キーワード:</p>
+  <input type="search" id="keywords">
+  <button onclick="keywords()"><img src="../assets/検索_mock.png" width="25" height="25"/></button>
+  </div>
+  <button onclick="setting()" id="setting">設定</button>
+  <button onclick="newnote()" id="newnote">新規ノートを追加</button>
+  </div><br>
+
+  <div class="sort">
+  <button onclick="datesort()" id="datesort">日時順でソート</button>
+  <button onclick="titlesort()" id="titlesort">タイトル順でソート</button>
+  </div>
 </template>
 
 <script setup lang="ts">
 
+
+
 </script>
+
+<style>  
```

Comment:
これは言ってなかった僕が悪いんですが、
```suggestion
<style scoped>
```
としてほしいです！

style に scoped 属性がないと、他コンポーネントに影響を与える可能性があるので！